docs/source/reference/io_descriptors.rst

```diff
@@ -0,0 +1,50 @@
+IO Descriptors
+======================
+EVA defines three main data types. The inputs and outputs from the UDFs must be of any of these types.
```

Comment:
Are there only two data types below? Or maybe I miss some thing.

Reply:
added the documentation for pandas dataframe.